src/scene_battle_rpg2k.cpp

```diff
@@ -450,12 +461,12 @@ bool Scene_Battle_Rpg2k::ProcessBattleAction(Game_BattleAlgorithm::AlgorithmBase
 			}
 
 			if (action->TargetNext()) {
-				battle_action_state = BattleActionState_Start;
+				battle_action_state = BattleActionState_ConditionHeal;
 				return false;
 			}
 
 			// Reset variables
-			battle_action_state = BattleActionState_Start;
+			battle_action_state = BattleActionState_ConditionHeal;
```

Comment:
Yeah i needed to change them to get poisen effect working before the battlers attac